src/dynamic-converter.ts

```diff
@@ -59,8 +59,16 @@ function delegatingConverter<R, V>(
         options
       );
     },
-    hasChange(currentValue: V, newValue: V): boolean {
-      return currentValue !== newValue;
```

Comment:
This was a bug in the initial PR, we should pass down the hasChange event to the child converter.
Re-enable this line and see the bug in the additional testcase